elements/rh-switch/demo/reversed.html

```diff
@@ -0,0 +1,41 @@
+<section>
+  <form>
+    <fieldset>
+      <legend>Reversed</legend>
+      <label for="reversed">
+        <span data-state="on">Message when on</span>
+        <span data-state="off" hidden>Message when off</span>
+      </label>
+      <rh-switch id="reversed" reversed></rh-switch>
```

Comment:
Is the reversed boolean attribute strictly necessary?

Reply:
There is no boolean attribute on `rh-switch`  so yes that needs removed.  

This demo was copied over from `pf-switch`, it doesn't exist there either, so should fix there as well.  Not even sure we need this demo at all.  The only thing it shows is that html order of label doesn't matter which is true with any form element as long as it uses a `for` attribute.  

I'll remove the attr for now but maybe we just delete the demo in full.